src/components/common/UploadDocumentEwallet.tsx

```
@@ -1,108 +1,467 @@
+
+		try {
+			// Only use postMessage for communication
+			iframeRef.current.contentWindow?.postMessage(messageData, VITE_EWALLET_IFRAME_SRC);
+		} catch (error) {
+			console.error('Failed to send message to iframe:', error);
+			setError('Failed to communicate with wallet. Please try again.');
+		}
+	};
+
+	// Prepare payload for document upload
+	const preparePayload = async (data: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)